staging/src/k8s.io/client-go/tools/cache/delta_fifo.go

```diff
@@ -46,6 +46,20 @@ import (
 // TODO(lavalamp): I believe there is a possible race only when using an
```

Comment:
Thanks for combining-- maybe we can change this to "NOTE: It is possible to misuse this and cause a race." It's not really a TODO anymore?

Reply:
Updated.